src/memory/memory.c

```diff
@@ -282,7 +282,7 @@ void memory_get_seed_birthdate(uint32_t* timestamp_out)
     }
 }
 
-bool memory_setup(memory_interface_functions_t* ifs)
+bool memory_setup(const memory_interface_functions_t* ifs)
```

Comment:
since you're here, there's also test_memory.c: https://github.com/benma/bitbox02-firmware/blob/5715aeb2/test/unit-test/test_memory.c#L110